core/src/main/scala/com/salesforce/op/stages/impl/selector/DefaultSelectorParams.scala

```diff
@@ -63,7 +63,14 @@ object DefaultSelectorParams {
   //    - "gamma"    : "inverse", "identity", "log"
   //    - "tweedie"  : power link function specified through "linkPower". The default link power in
   //    the tweedie family is 1 - variancePower.
-  val NumRound = Array(100) // number of rounds for xgboost (default 1)
-  val Eta = Array(0.1 , 0.3) // step size shrinkage for xgboost (default 0.3)
-  val MinChildWeight = Array(1.0, 5.0, 10.0) // minimum sum of instance weight needed in a child for xgboost (default 1)
+  val NumRound = Array(200) // number of rounds for xgboost (default 1)
+  val Eta = Array(0.02) // step size shrinkage for xgboost (default 0.3)
+  val MinChildWeight = Array(1.0, 10.0) // minimum sum of instance weight needed in a child for xgboost (default 1)
+  val MaxDepthXGBClassifier = Array(10) // max depth of each tree for xgboost (default is 6)
```

Comment:
Can you make the variable names standard? Right now, some have "Classifier" at the end and some have "BinaryClass" and the beginning.